src/lib/profiles/data-management/Current/LoggingManagement.h

```diff
@@ -292,6 +302,7 @@ class LoggingManagement
     uint32_t mThrottled;
     ImportanceType mMaxImportanceBuffer;
     bool mUploadRequested;
+    bool mShutdownInProgress;
```

Comment:
I don't see any purpose for this field.  Why do we need a "shutdown in progress" state?  And why isn't this another value in the LoggingManagementStates enum?